# <a href="https://www.pythontutorial.net/advanced-python/python-rounding/" style="color:Tomato">Python Rounding</a>

Trong bài này, ta sẽ học về cách sử dụng hàm `round()` trong Python để làm tròn số.

### Tables of Contents
* [Introduction to the Python `round()` function](#1)
* [Python `round()` function examples](#2)
* [Summary](#sum)

## <a class="anchor" id="1">Introduction to the Python `round()` function</a>

Để làm tròn số trong Python, ta sử dụng hàm `round()`:
```python
round(number [, ndigits])
```

trong đó:
- `number`: Đây là số thực bạn muốn làm tròn.
- `ndigits`: Đây là số nguyên thể hiện số chữ số thập phân sau dấu phẩy trong kết quả. Nếu không cung cấp `ndigits`, số sẽ được làm tròn thành một số nguyên.

## <a class="anchor" id="2">Python `round()` function examples</a>

In [1]:
round(1.25)

1

In [2]:
round(1.25, 0)

1.0

In [3]:
round(15.5, -1)

20.0

In [4]:
round(1.25, 1)

1.2

In [5]:
round(1.35, 1)

1.4

Chờ đã, tại sao làm tròn `1.25` thì lại được `1.2` mà làm tròn `1.35` thì lại được `1.4`?

Ví dụ xét dòng code `round(1.25, 1)`, ta đang muốn làm tròn số `1.25` thành số gần nhất có 1 chữ số thập phân, nhưng ở đây có 2 số thoả mãn là `1.2` và `1.3`.

Trong trường hợp này, ta áp dụng "*banker’s rounding*", nghĩa là chọn số có chữ số ý nghĩa cuối cùng là số chẵn.

## <a class="anchor" id="sum" style="color:Violet"> Tổng kết </a>

Sử dụng hàm `round(number, ndigits)` để làm tròn số `number` đến `ndigits` chữ số thập phân sau dấu phẩy.